dev-docs/bidders/equativ.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: bidder
+title: Equativ
+description: Prebid Equativ Bidder Adapter
+biddercode: equativ
+media_types: banner
+tcfeu_supported: true
+coppa_supported: true
+gvl_id: 45
+gpp_supported: true
```

Comment:
This is deprecated. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for the newer `gpp_sids` metadata.